manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager

```

```suggestion
PackageIdentifier: Autodesk.DownloadManager
```
The publisher already exists as `Autodesk` so the package ID here should be updated to match